doc/other/other.md

```diff
@@ -0,0 +1,14 @@
+# Other Modules
+
+```{eval-rst}
+.. toctree::
+    :hidden:
+    
+    Util           <util.md>
+    Widgets        <widgets.md>
```

Comment:
these are part of flame-core, should they be on the main flame section?

Reply:
Also debugging. They could be under flame > other
but then this "other" means: other packages

Reply:
Sure, I can move those.